### PR TITLE
Issue 989 batch-2

### PR DESCRIPTION
**Pull request by hbothra15**
In this batch I have update all project having name started with B and C

- Update all the pom files, need to pay special attention to commander app's pom file which generate multiple jar in single go. This approach was used as this project has multiple main class.
- It resolves partially #issue-989

### CODE REVIEW

**Comment by hbothra15** [comment]
Hi @iluwatar,
Sorry man for the delay, will try to update the batches on weekly basis, as need some office work to deal with.

Regards
@hbothra15

**Comment by iluwatar** [comment]
@hbothra15 please resolve the conflicts

File: eip-message-channel/pom.xml [hunk 1]
```diff
@@ -48,4 +48,23 @@
...
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-assembly-plugin</artifactId>
+				<executions>
+					<execution>
+						<configuration>
+							<archive>
+								<manifest>
+									<mainClass>com.iluwatar.epi.message.channel.App</mainClass>
```
Comment (iluwatar) [inline eip-message-channel/pom.xml]: I think there's a typo here: `epi` -> `eip`

**Comment by iluwatar** [comment]
Thanks @hbothra15 for this batch :+1:

**Pull request merged**